### PR TITLE
Dragging movement understanding - Sarah's update

### PR DESCRIPTION
**Pull request by alastc**

### CODE REVIEW

File: understanding/22/dragging-movements.html [hunk 1]
```diff
@@ -37,19 +37,21 @@ <h2>Intent of Dragging Movements</h2>
...
-      <p>In contrast to dragging movements, path-based gestures involve at least an initial directionality of the pointer movement. This is defined by stating that the path has to go through at least one intermediate point to qualify as pointer gesture. The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined. For more details, refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">Understanding Success Criterion 2.5.1 Pointer Gestures</a>.</p>
+      
+      <p>Dragging movements covered in this Success Criterion are pointer interactions where only the endpoints matter. Once the pointer engages with a target, the direction of the pointer motion does not factor into the interaction until the pointer disengages that target. Dragging is a motion between the start and end points without an intermediate point. The motion can go in any direction and is entirely undefined. </p>
```
Comment (detlevhfischer) [inline understanding/22/dragging-movements.html:50]: This paragraph seems a bit redundant, there two similar statements about dragging motions (we also need tosettle for eirher ‚movement‘ or ‚motion‘)

**Review by detlevhfischer — APPROVED** [review]
Agree with changes overall - but see comment about redundancy in one para.

File: understanding/22/dragging-movements.html (hunk 1 above)
Comment (gradualclearing) [reply understanding/22/dragging-movements.html]: See revision for reducing redundancy.

Regarding dragging movement versus draggin motion, a dragging movement seems like what is defined and provided by the system and a dragging motion is what is performed by the user. Does that make sense? I can take a pass through the whole document and update to meet these definitions.

**Comment by alastc** [comment]
Approved: https://www.w3.org/2021/03/16-ag-minutes.html#item03

**Pull request merged**